src/tagmanager/tm_ctags.c

```diff
@@ -241,6 +241,29 @@ void tm_ctags_clear_ignore_symbols(void)
 }
 
 
+static void replace_str(gchar **where, const gchar *what, guint what_len,
+	const gchar *replacement, guint replacement_len)
+{
+	if (where && *where)
+	{
+		guint where_len = strlen(*where);
+		gchar *pos = strstr(*where, what);
+
+		if (pos)
+		{
+			gchar *str = g_malloc(where_len + replacement_len);
+			guint prefix_len = pos - *where;
```

Comment:
conversion: pointer difference yields signed (although we know it's `>= 0` in this case indeed), so we "need" an explicit cast

Reply:
OK.